### ISSUE TITLE
Exception Value: 'NoneType' object has no attribute '_inc_path', if plugin has more than one child

### ISSUE THREAD

**Issue by JurajKavka** [divio/django-cms#6011]
### Summary
In the fresh installed Django CMS project (installed "by hand") I have only base template with one placeholder "content". Each attempt to add bootstrap3 `Row` plugin with more than two columns ends with error `'NoneType' object has no attribute '_inc_path'`.

Running `python2 manage.py cms fix-tree` does not help.

#### Environment

* Python version: 2.7.13
* Django version: 1.9
* django CMS version: 3.4.4

### Traceback
```
Environment:


Request Method: POST
Request URL: http://localhost:8000/sk/admin/cms/page/add-plugin/?placeholder_id=4&plugin_type=Bootstrap3RowCMSPlugin&cms_path=%2Fen%2Fest%2F&plugin_language=en

Django Version: 1.9
Python Version: 2.7.13
Installed Applications:
['djangocms_admin_style',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.sites',
 'django.contrib.staticfiles',
 'cms',
 'menus',
 'treebeard',
 'sekizai',
 'filer',
 'easy_thumbnails',
 'mptt',
 'djangocms_text_ckeditor',
 'aldryn_bootstrap3',
 'core']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'cms.middleware.user.CurrentUserMiddleware',
 'cms.middleware.page.CurrentPageMiddleware',
 'cms.middleware.toolbar.ToolbarMiddleware',
 'cms.middleware.language.LanguageCookieMiddleware',
 'cms.middleware.utils.ApphookReloadMiddleware']



Traceback:

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  149.                     response = self.process_exception_by_middleware(e, request)

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  147.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/django/utils/decorators.py" in _wrapped_view
  149.                     response = view_func(request, *args, **kwargs)

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
  57.         response = view_func(request, *args, **kwargs)

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/django/contrib/admin/sites.py" in inner
  244.             return view(request, *args, **kwargs)

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/django/views/decorators/clickjacking.py" in wrapped_view
  39.         resp = view_func(*args, **kwargs)

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/cms/admin/placeholderadmin.py" in add_plugin
  367.         response = plugin_instance.add_view(request)

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/django/contrib/admin/options.py" in add_view
  1435.         return self.changeform_view(request, None, form_url, extra_context)

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/django/utils/decorators.py" in _wrapper
  67.             return bound_func(*args, **kwargs)

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/django/utils/decorators.py" in _wrapped_view
  149.                     response = view_func(request, *args, **kwargs)

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/django/utils/decorators.py" in bound_func
  63.                 return func.__get__(self, type(self))(*args2, **kwargs2)

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/django/utils/decorators.py" in inner
  184.                     return func(*args, **kwargs)

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/django/contrib/admin/options.py" in changeform_view
  1378.                 self.save_model(request, new_object, form, not add)

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/aldryn_bootstrap3/cms_plugins.py" in save_model
  83.             col.save()

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/cms/models/pluginmodel.py" in save
  333.                 self.parent.add_child(instance=self)

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/treebeard/mp_tree.py" in add_child
  1013.         return MP_AddChildHandler(self, **kwargs).process()

File "/home/juro/work/prog/web/dj/weblso/.env/lib/python2.7/site-packages/treebeard/mp_tree.py" in process
  387.             newobj.path = self.node.get_last_child()._inc_path()

Exception Type: AttributeError at /sk/admin/cms/page/add-plugin/
Exception Value: 'NoneType' object has no attribute '_inc_path'
```

**Comment by czpython** [comment]
Hello @JurajKavka,
Thanks for reporting this.
I believe the issue is related to how bootstrap3 creates the child plugins.
I've closed the ticket here and opened one in https://github.com/aldryn/aldryn-bootstrap3/issues/155

Can you add a comment on that ticket with the database you're using?

**Comment by JurajKavka** [comment]
I made further tests. I have started project with `djangocms -f -p . mysite` with `aldryn-bootstrap3` plugin. Everything works fine with sqlite database. When using the Postgres I can't create row with two cols. Also I can't create two `djangocms_column` in one placeholder.

Using the postgres i also facing issue during publishing page.

```
IntegrityError at /en/admin/cms/page/1/en/publish/
duplicate key value violates unique constraint "cms_cmsplugin_path_7692c19a7d5df9d5_uniq"
DETAIL:  Key (path)=(000A) already exists.
```
I know it's anoher and often reported topic but it is... With sqlite - no problem.
